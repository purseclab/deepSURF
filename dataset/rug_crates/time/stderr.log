    Finished test [unoptimized + debuginfo] target(s) in 0.02s
     Running unittests src/lib.rs (target/debug/deps/time-76c9c361b219fa85)
     Running ../tests/main.rs (target/debug/deps/tests-902b05b3708a7d65)
    Finished test [unoptimized + debuginfo] target(s) in 0.03s
     Running unittests src/lib.rs (/home/xiang/workspace/fuzz2_base/time/target/debug/deps/time-8886e69373111e63)
     Running ../tests/main.rs (/home/xiang/workspace/fuzz2_base/time/target/debug/deps/tests-8249b60609e189ea)
   Doc-tests time
   Doc-tests time
