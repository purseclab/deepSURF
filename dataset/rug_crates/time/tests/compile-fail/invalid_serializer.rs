use time::serde;

serde::format_description!(); // unexpected end of input
serde::format_description!("bad string", OffsetDateTime, "[year] [month]"); // module name is not ident
serde::format_description!(my_format: OffsetDateTime, "[year] [month]"); // not a comma
serde::format_description!(my_format,); // missing formattable and string
serde::format_description!(my_format, "[year] [month]"); // missing formattable
serde::format_description!(OffsetDateTime, "[year] [month]"); // missing ident
serde::format_description!(my_format, OffsetDateTime); // missing string format
serde::format_description!(my_format, OffsetDateTime,); // missing string format
serde::format_description!(my_format, OffsetDateTime "[year] [month]"); // missing comma
serde::format_description!(my_format, OffsetDateTime : "[year] [month]"); // not a comma
serde::format_description!(my_format, OffsetDateTime, "[bad]"); // bad component name
serde::format_description!(my_format, OffsetDateTime, not_string); // string format wrong type

fn main() {}
