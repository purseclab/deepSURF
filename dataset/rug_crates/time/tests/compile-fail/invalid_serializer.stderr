error: unexpected end of input
 --> ../tests/compile-fail/invalid_serializer.rs:3:1
  |
3 | serde::format_description!(); // unexpected end of input
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `serde::format_description` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected token: "bad string"
 --> ../tests/compile-fail/invalid_serializer.rs:4:28
  |
4 | serde::format_description!("bad string", OffsetDateTime, "[year] [month]"); // module name is not ident
  |                            ^^^^^^^^^^^^

error: unexpected token: :
 --> ../tests/compile-fail/invalid_serializer.rs:5:37
  |
5 | serde::format_description!(my_format: OffsetDateTime, "[year] [month]"); // not a comma
  |                                     ^

error: unexpected end of input
 --> ../tests/compile-fail/invalid_serializer.rs:6:1
  |
6 | serde::format_description!(my_format,); // missing formattable and string
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `serde::format_description` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected token: "[year] [month]"
 --> ../tests/compile-fail/invalid_serializer.rs:7:39
  |
7 | serde::format_description!(my_format, "[year] [month]"); // missing formattable
  |                                       ^^^^^^^^^^^^^^^^

error: unexpected token: "[year] [month]"
 --> ../tests/compile-fail/invalid_serializer.rs:8:44
  |
8 | serde::format_description!(OffsetDateTime, "[year] [month]"); // missing ident
  |                                            ^^^^^^^^^^^^^^^^

error: unexpected end of input
 --> ../tests/compile-fail/invalid_serializer.rs:9:1
  |
9 | serde::format_description!(my_format, OffsetDateTime); // missing string format
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `serde::format_description` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected end of input
  --> ../tests/compile-fail/invalid_serializer.rs:10:1
   |
10 | serde::format_description!(my_format, OffsetDateTime,); // missing string format
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = note: this error originates in the macro `serde::format_description` (in Nightly builds, run with -Z macro-backtrace for more info)

error: unexpected token: "[year] [month]"
  --> ../tests/compile-fail/invalid_serializer.rs:11:54
   |
11 | serde::format_description!(my_format, OffsetDateTime "[year] [month]"); // missing comma
   |                                                      ^^^^^^^^^^^^^^^^

error: unexpected token: :
  --> ../tests/compile-fail/invalid_serializer.rs:12:54
   |
12 | serde::format_description!(my_format, OffsetDateTime : "[year] [month]"); // not a comma
   |                                                      ^

error: invalid component
  --> ../tests/compile-fail/invalid_serializer.rs:13:55
   |
13 | serde::format_description!(my_format, OffsetDateTime, "[bad]"); // bad component name
   |                                                       ^^^^^^^

error[E0432]: unresolved import `not_string`
  --> ../tests/compile-fail/invalid_serializer.rs:14:1
   |
14 | serde::format_description!(my_format, OffsetDateTime, not_string); // string format wrong type
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ no external crate `not_string`
   |
   = note: this error originates in the macro `serde::format_description` (in Nightly builds, run with -Z macro-backtrace for more info)
